cmd/templates-api/error.go

```diff
@@ -0,0 +1,220 @@
+// nolint: errorlint
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"runtime/debug"
+	"strings"
+
+	"github.com/iancoleman/strcase"
+	goaHTTP "goa.design/goa/v3/http"
+	"goa.design/goa/v3/middleware"
+	goa "goa.design/goa/v3/pkg"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/json"
+	"github.com/keboola/keboola-as-code/internal/pkg/template/api/dependencies"
+	"github.com/keboola/keboola-as-code/internal/pkg/template/api/gen/templates"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/strhelper"
+)
+
+const (
+	ExceptionIdPrefix   = "keboola-templates-"
+	ErrorNamePrefix     = "templates."
+	DefaultErrorName    = "internalError"
+	DefaultErrorMessage = "Application error. Please contact our support support@keboola.com with exception id (%s) attached."
+)
```

Comment:
V tomto subore je vsetko ohladom spracovania chyby.
Je to inspirovane ostatnymi APIs.